### PR TITLE
fix: fixes error when there are conflicting network names

### PR DESCRIPTION
**Pull request by andrewjschuang**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
 Bug fix.

* **What is the current behavior?** (You can also link to an open issue here)
 Flowbroker couldn't find the correct network if there were two or more networks containing 'flowbroker' in their names.

* **What is the new behavior (if this is a feature change)?**
 The network name has to be defined by the user and, being each one unique, there won't be any problems in finding the correct one.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
 No.

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
This is connected to dojot/dojot#665.

* **Other information**:
This is connected to dojot/docker-compose#74.

### CODE REVIEW

File: orchestrator/nodes/dockerComposeRemoteNode/index.js [hunk 1]
```diff
@@ -48,13 +48,20 @@ class DataHandler extends RemoteNode {
       }
 
       this.client.networks().list().then((results) => {
-        for (let result of results) {
-          let name = result.Name.match(/.+?_flowbroker/);
-          if (name) {
-            this.targetNetwork = name;
-            return resolve(name);
+        let network = process.env.FLOWBROKER_NETWORK;
```
Comment (raulnegreiros) [inline orchestrator/nodes/dockerComposeRemoteNode/index.js]: I think that should be interesting add this environment variable on 'config.js' file.
When env variables are just used in somewhere in the code become difficult to have an overview that which environment variables are requested.

Comment (raulnegreiros) [inline orchestrator/nodes/dockerComposeRemoteNode/index.js]: I would prefer that the network information become a class attribute and the assignment occurs on class constructor.
The reason is class design and code maintainability.

Comment (andrewjschuang) [reply orchestrator/nodes/dockerComposeRemoteNode/index.js]: What do you think about it now?

Comment (raulnegreiros) [reply orchestrator/nodes/dockerComposeRemoteNode/index.js]: seems fine.

**Pull request merged**